src/OMSimulatorLib/Model.cpp

```diff
@@ -378,6 +379,11 @@ oms_status_enu_t oms::Model::exportSnapshot(const oms::ComRef& cref, char** cont
     updateParameterBindingsToSSD(system_node, ssvNode, true);
     // TODO ssm file
   }
+  else
+  {
+    // delete ssv node from <oms:snapshot> if flag not set
+    snapshot.deleteResourceNode("resources/" + std::string(this->getCref()) + ".ssv");
+  }
```

Comment:
Why do we generate the node in the first place?

Reply:
this is because of the current signature of `exportToSSD(pugi::xml_node& node, pugi::xml_node& ssvNode, Snapshot& snapshot)`, And this should be changed when we restructure the `exportToFile` to use `snapshot` interface